Packaging/fedora/devilutionx.spec

```diff
@@ -0,0 +1,50 @@
+%define debug_package %{nil}
+
+Name:		devilutionx
+Version:	0.3.1
+Release:	1%{?dist}
+Summary:	Diablo I engine for modern operating systems
+
+License:	Unlicensed
+URL:		https://github.com/diasurgical/devilutionX
+Source0:	https://github.com/diasurgical/devilutionX/archive/%{version}.tar.gz
+Source1:	devilutionx.desktop
+
+BuildRequires:	cmake gcc gcc-c++ libstdc++-static glibc desktop-file-utils
+BuildRequires:  glibc-devel SDL2-devel SDL2_ttf-devel SDL2_mixer-devel libsodium-devel libasan
+Requires:	SDL2_ttf SDL2_mixer libsodium
+
+%description
+Diablo devolved - magic behind the 1996 computer game
+Note, Devilution requires an original copy of diabdat.mpq. None of the Diablo 1 game assets are provided by this package. 
+
+%prep
+%setup -q -n devilutionX-%{version}
+
+
+%build
+cmake -DBINARY_RELEASE=ON -DDEBUG=OFF
+make %{?_smp_mflags}
+
+%install
+make INSTALL_ROOT=%{buildroot}
+#according to Fedora's Games Packaging guidelines the binary should go in %{_bindir}
+#However, the problem is that the mpq files do actually not belong in this directory.
+#That's why we gonna use /var/games/%{name} for the moment, but this needs to be addressed though.
```

Comment:
The game will also look for assets in ~/.local/share/diasurgical/devilution/

Reply:
Oh, ok, didn't know that, but perhaps it's still possible to implement a switch for the datadir?

Reply:
Yeah eventually we need to change how data is loaded, this is currently happening in the system abstraction portion of the code but doing it this way is a bit limiting, for now we are trying to touch the actual engine as litter as possible as there is still a lot of cleaning happening upstream and we would like to keep merge conflicts to a minimum.

Reply:
No worries.
I wasn't demanding such a switch but I just think that it would be a "nice to have feature".